TidepoolOnboardingKitUI/View Models/OnboardingViewModel.swift

```diff
@@ -31,14 +32,21 @@ class OnboardingViewModel: ObservableObject, CGMManagerCreateNotifying, CGMManag
 
         self.sectionProgression = onboarding.sectionProgression
         self.therapySettings = onboarding.therapySettings
+        self.dosingEnabled = onboarding.dosingEnabled ?? true
```

Comment:
[nit] looks like it defaults to true

Reply:
Yes, in the view model it does because that is the default for the view itself when it displays. However, the unset state (`nil`) is correct for the overall onboarding object (since it indicates it hasn't been displayed in the UI yet).